README.md

```diff
@@ -86,6 +91,44 @@ Specifically, these workflows do not address provenance
 [verification](https://slsa.dev/spec/v1.0/verifying-artifacts). Those requirements
 must be handled separately to meet SLSA Build level 3+.
 
+### Hall of Fame
+
+#### Generation of Provenance
+
+Below is a non exhaustive list of projects that use the builders in this repository to generate provenance:
+
+1. [flask](https://github.com/pallets/flask) ![flask stars](https://img.shields.io/github/stars/pallets/flask)
```

Comment:
nit: If you add the repo name as a label and github as a logo you could combine the repo name and stars badge.

Something like this
[![flask stars](https://img.shields.io/github/stars/pallets/flask?logo=github&label=pallets/flask)](https://github.com/pallets/flask)

```suggestion
1. [![flask stars](https://img.shields.io/github/stars/pallets/flask?logo=github&label=pallets/flask)](https://github.com/pallets/flask)
```

Reply:
great, I'll do that!